policyengine_us_data/datasets/cps/cps.py

```diff
@@ -229,8 +229,23 @@ def add_personal_variables(cps: h5py.File, person: DataFrame) -> None:
         person.A_AGE == 80,
         # NB: randint is inclusive of first argument, exclusive of second.
         np.random.randint(80, 85, len(person)),
-        person.A_AGE,
+        np.where(
```

Comment:
How about

```python
def _assign_some_newborns_to_pregnancy(age: pd.Series, percent_newborns: 0.5) -> pd.Series:
""" Takes an array of ages, returns the new age array with the given percentage of newborns assigned a negative age (in pregnancy)."""
  age = np.where(...)
  return age
```

Then here, leave code as it was and add:
```python
cps["age"] = _assign...(cps["age"])
```

If that breaks try `cps["age"][...]` over `cps["age"]`.